src/pages/Transfer/CrossChainTransferForm/CrossChainTransferForm.tsx

```diff
@@ -109,9 +109,27 @@ const CrossChainTransferForm = (): JSX.Element => {
     validateOnChange: false
   });
 
+  const getTokenData = async () => {
```

Comment:
`setTokenData`, since you not only get but also set. Better make it `useCallback`